src/components/menu/__tests__/editor-mode-toolbar.test.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable import/no-named-as-default */
+import React from 'react'
+import { shallow } from 'enzyme'
+import { EditorModeToolbarUnconnected, mapStateToProps } from '../editor-mode-toolbar'
+import EditorModeTitle from '../editor-mode-title'
+import ViewControls from '../view-controls'
+import IodideLogo from '../../../shared/iodide-logo'
+import EditorModeControls from '../editor-mode-controls'
+
+describe('EditorModeToolbarUnconnected', () => {
+  let mountedPane
+  let props
+  const wrapper = () => {
+    if (!mountedPane) {
```

Comment:
maybe change `mountedPane` -> `mountedComponent`, which is a bit more generic.
(in this case the component we're testing is not a pane)

Reply:
Sorry I missed it. I made the changes.